packages/client/src/components/Layout.vue

```diff
@@ -1,43 +1,63 @@
 <template>
   <div>
     <b-navbar type="dark" variant="dark">
-      <b-navbar-brand href="/#/grants">Grants Identification Tool</b-navbar-brand>
+      <b-navbar-brand href="/#/grants"
+        >Grants Identification Tool</b-navbar-brand
```

Comment:
There are some weird formatting changes in this diff, do you have some kind of editor integration doing this? Can you update the PR before merging to only contain the relevant changes (which looks like just one line right now)?